backend/src/routes/results.js

```diff
@@ -1,6 +1,20 @@
 import express from 'express';
+import Result from '../mongo/models/Result';
 const router = express.Router();
 
-// TODO: Handlers...
+// GET: Get results
+router.get('/results/:id', async (req, res) => {
```

Comment:
Your path doesn't need to have results in it since this is already defined in index.js